components/Sequence/sequence_events.rst

```diff
@@ -0,0 +1,12 @@
+Sequence Events
```

Comment:
It was discussion about events. I am not sure whether we should describe them. WDYT @pjedrzejewski.